scenes/game/game.gd

```diff
@@ -1883,28 +1922,30 @@ func begin_gauge_strike_choosing(strike_response : bool, cancel_allowed : bool,
 			new_sub_state = UISubState.UISubState_SelectCards_StrikeCard_FromSealed
 	change_ui_state(UIState.UIState_SelectCards, new_sub_state)
 
-func begin_boost_choosing(can_cancel : bool, allow_gauge : bool, only_gauge : bool, limitation : String):
+func begin_boost_choosing(can_cancel : bool, valid_zones : Array, limitation : String, ignore_costs : bool):
 	selected_cards = []
 	select_card_require_min = 1
 	select_card_require_max = 1
-	select_boost_from_gauge = allow_gauge
-	select_boost_only_from_gauge = only_gauge
+	select_boost_valid_zones = valid_zones
 	select_boost_limitation = limitation
+	select_boost_ignore_costs = ignore_costs
```

Comment:
we don't need all these as member variables on game.gd and boost_selection_options, right?

Reply:
I could probably convert all of them to one or the other; would the dictionary or member variables be more convenient?